### DISCUSSION TITLE
Change in "etc/resolv.conf" file on Fedora upgrade ! Does need for revert ?

### DISCUSSION THREAD

**Discussion by Nokia808** [DNSCrypt/dnscrypt-proxy#2313] [Q&A]
Hello !

I'm chronic user for this excellent program. I'm on Fedora since Fedora 24 & upgraded my Fedora using command line every 1 year. Currently I'm on Fedora 36 X 64 bit after upgraded from Fedora 34 to 36.

I'm already set the following parameters in "/etc/resolv.conf" as following:

```
nameserver 127.0.0.1
options edns0
```

But, yesterday, I checked "/etc/resolv.conf" & saw the following parameters:

```
nameserver 127.0.0.53
options edns0  trust-ad
```

see bellow screenshot:
![screenshot](https://user-images.githubusercontent.com/25375330/217538280-dbc24582-4f61-4cdc-8c6f-22a262503d4e.png)

It seem that change is a result from upgrading my distro !

Now what I have to do now ?? Re-edit "/etc/resolv.conf" file again (revert "127.0.0.53" into "127.0.0.1" & remove "trust-ad") or I should leave every thing as it ??

Thank you, & sorry for bothering you.

**Comment by jedisct1** [comment]
It should be fine to leave it.

`trust-ad` is something that probably wasn't supported in the previous version of your distro, and now it is, so it was added automatically.

**Comment by Nokia808** [comment]
Thank you for your kind response.

Just 1 further point to be sure 100% from my current setting: I'm already set - since years ago - my network manager GUI with "127.0.0.1" - aee screenshot bellow:

![screenshot-gui](https://user-images.githubusercontent.com/25375330/218035458-527de68f-f5a1-431b-b95c-a6fb6373b53b.png)


Is there a need to change it to "127.0.0.53" or I should leave it as it ?
